### PR TITLE
[luxtronik] Add parameter for fixed heating return temperature

### PR DESCRIPTION
**Pull request by Taxifahrer**
I run my Apha Innotec LWC heat pump in fixed temperature mode, so I can directly control the target heating return temperature via openHAB (higher temperature if solar is generating enough power, etc.). Currently upgrading from OH2, where I had made private changes to the addon, so maybe we can add this to the standard code.

### CODE REVIEW

File: bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/i18n/luxtronikheatpump_de.properties [hunk 1]
```diff
@@ -240,6 +240,7 @@ channel-type.luxtronikheatpump.temperatureCondensation.label = Kondensationstemp
 channel-type.luxtronikheatpump.temperatureExhaustAir.label = Ablufttemp.
```
Comment (lsiepel) [inline bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/i18n/luxtronikheatpump_de.properties:240]: Please remove this file, as translations are made available by crowdin. Only the defualt translation file should be in the PR.

**Review by lsiepel — CHANGES_REQUESTED** [review]
As you are adding a channel, you also need to add thing-upgrade instructions. And the channel is not yet added to thing-types.xml

**Review by sgiehl — CHANGES_REQUESTED** [review]
I'm not able to test that, as my heatpumpt doesn't run in that mode.
Also maybe we should use `temperatureHeatingFixedReturnTarget`, so it is a bite more meaningfull maybe?

File: bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml [hunk 2]
```diff
@@ -2059,6 +2059,13 @@
 		<state pattern="%.1f %unit%" min="30" max="65" step="0.5"></state>
 	</channel-type>
 
+	<channel-type id="temperatureHeatingFixedReturnTarget">
```
Comment (lsiepel) [inline bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml:2062]: Don;t want to be a party pooper here. Not sure how strict the maintainers hold on to the namingconvention as this binding is allready full of the old method, but it should be `temperature-heating-fixed-return-target` or similar.

Comment (Taxifahrer) [reply bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml:2062]: I probably shouldn't break this by cleaning up the naming convention for all current users. But I also don't think I should do this one different than the others. What do you suggest?

File: bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/update/instructions.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
```
Comment (lsiepel) [inline bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/update/instructions.xml:1]: Great, now only the `thingTypeVersion` has to be set in thing-types.xml
```
  <properties>
    <property name="thingTypeVersion">1</property>
  </properties>
```

Refernece: https://www.openhab.org/docs/developer/bindings/thing-xml.html#updating-thing-types

File: bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml (hunk 2 above)
Comment (lsiepel) [reply bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml:2062]: @openhab/add-ons-maintainers have to decide.

Comment (jlaur) [reply bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml:2062]: As a general rule the [naming convention](https://www.openhab.org/docs/developer/guidelines.html#naming-convention) only applies to new bindings. Existing bindings should stick to the naming convention already used by the binding to keep consistent naming within the binding.

File: bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/update/instructions.xml (hunk 3 above)
Comment (Taxifahrer) [reply bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/update/instructions.xml:1]: Done.

**Review by lsiepel — APPROVED** [review]
LGTM

**Comment by Taxifahrer** [comment]
I tested it in my installation, and it works (receive updates, and changing the value).
Is there anything else I need to do to get it merged?

**Comment by sgiehl** [comment]
@Taxifahrer Looks like the DCO check is failing. You might need to rebase your branch and ensure all commits are signed-off

**Review by bern77 — COMMENTED** [review]
LGTM for HeliosEasyControls

**Pull request closed without merging**